### PR TITLE
4.4.5

### PR DESCRIPTION
**Pull request by ericgaspar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by ericgaspar** [comment]
!testme

**Comment by mrflos** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by mrflos** [comment]
!testme again

**Comment by mrflos** [comment]
!testme

**Comment by ericgaspar** [comment]
!testme

**Comment by Lab-8916100448256** [comment]
!testme please, pretty please, with a cherry on top

**Comment by Lab-8916100448256** [comment]
!testme
please, pretty please, with a cherry on top

File: scripts/backup [hunk 1]
```diff
@@ -13,32 +12,25 @@ ynh_print_info "Declaring files to be backed up..."
...
-#=================================================
-
-ynh_backup "/etc/sudoers.d/$app"
-
-#=================================================
-# BACKUP THE IMPORTERS HOOKS
+# BACKUP VARIOUS FILES
 #=================================================
 
+if [ -d "/etc/sudoers.d/$app" ]; then
```
Comment (Lab-8916100448256) [inline scripts/backup:26]: this should be `if [ -f ...`
/etc/sudoers.d/$app is a file not a directory

**Review by Lab-8916100448256 — CHANGES_REQUESTED** [review]

**Pull request merged**